src/tools/material.js

```diff
@@ -39,7 +40,28 @@ function scaleNonPeriodicLattice(material) {
     return newLattice;
 }
 
+function getBasisConfigTranslatedToCenter(material) {
```

Comment:
The idea is that the tools are working on a material object and return a modified material. We can extract the config by running `toJSON` on `material.Basis`